src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -1592,6 +1592,21 @@ struct FFXIVIpcWardYardInfo :
   } object[100];
 };
 
+/**
+* Structural representation of the packet sent by the server
+* to show the current shared estate settings
+*/
+struct FFXIVIpcSharedEstateSettingsResponse :
+  FFXIVIpcBasePacket< SharedEstateSettingsResponse >
+{
+  struct playerEntry {
```

Comment:
For a second, i was afraid i wouldn't have anything to nag about...
new line here...